Documentation.md

```diff
@@ -164,590 +174,606 @@ and made available to your JS client code.
 * You have the option to show or hide the pre-built UI by showing or hiding the
 `containerDiv` into which you place the iframe, or applying a CSS rule like
 this:
-```
+```css
 .containerDiv iframe {
-   display: none;
+  display: none;
 }
 ```
 * The pre-built CCP UI is portrait oriented and capable of contracting
-  horizontally to fit smaller widths.  It can expand from a width of 200px to
-  a maximum of 320px based on the size of its container div.  If the CCP is
+  horizontally to fit smaller widths. It can expand from a width of 200px to
+  a maximum of 320px based on the size of its container div. If the CCP is
   placed into a container where it would be sized under 221px in width, the CCP
   switches to a different layout style with smaller buttons and fonts.
-* In its normal larger style, the CCP is 465px tall.  In its smaller form, the
+* In its normal larger style, the CCP is 465px tall. In its smaller form, the
   CCP is reduced to 400px tall.
 * CSS styles you add to your site will NOT be applied to the CCP because it is
   rendered in an iframe.
 * If you are trying to use chat specific functionalities, please also include
   [ChatJS](https://github.com/amazon-connect/amazon-connect-chatjs) in your code.
-  We omit ChatJS from the Makefile so that streams can be used without ChatJS. 
+  We omit ChatJS from the Makefile so that streams can be used without ChatJS.
   Streams only needs ChatJS when it is being used for chat. Note that when including ChatJS,
-  it must be imported after StreamsJS, or there will be AWS SDK issues 
+  it must be imported after StreamsJS, or there will be AWS SDK issues
   (ChatJS relies on the ConnectParticipant Service, which is not in the Streams AWS SDK).
 
-### Event Subscription
+### `connect.core.terminate()`
+
+Unitializes Amazon Connect Streams. Removing any subscription methods that have been called.
+
+The CCP iframe will not be removed though, so you'll have to manually remove it.
```

Comment:
We have a use case of a single page application that needs to re-initialize the CCP with another Amazon Connect instance. Hence, we need to "clean" all previous subscriptions.

I found this nice method which seemed to do the trick, so I decided to document it. However, I don't know if this method is intended to be used by customers.

Reply:
Thank you for doing this! Yes, we have had some customers use this in the past, and we should have documented it.